### DISCUSSION TITLE
Game of Throlls

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#471] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/game-of-throllnes/

**Comment by angellisy** [comment]
Adorei o *reskin*. Até mantiveram a carinha de depressivo do personagem :laughing:

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/videos/game-of-throllnes/#comment_1142146624>):

> Adorei o *reskin*. Até mantiveram a carinha de depressivo do personagem 😆

![null](<https://i.postimg.cc/nMb3mm8V/eating-popcorn-meme.gif>)
